sagemaker-datawrangler/joined-dataflow/explore_data.ipynb

```diff
@@ -0,0 +1,285 @@
+{
```

Comment:
<p>Use present tense "we perform preliminary data exploration to understand the data"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>